pkg/apis/dynatrace/v1alpha1/base_types.go

```diff
@@ -78,6 +79,12 @@ type BaseOneAgentSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Use immutable image"
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:advanced,urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	UseImmutableImage bool `json:"useImmutableImage,omitempty"`
+
+	// Optional: define resources requests and limits for single pods
```

Comment:
```suggestion
	// Optional: define resources requests and limits for the attached init container
```

You may need to have the field individually for OneAgent and OneAgentAPM (and to be fair, I wonder if we should rename it for the OneAgentAPM since they don't mean the same)